csharp/src/Ice/LocalException.cs

```diff
@@ -292,15 +299,43 @@ public InvalidDataException(string message, Exception innerException)
         }
     }
 
-    /// <summary>This is a purely Ice-internal exception used for retries.</summary>
-    public class RetryException : Exception
+    /// <summary>This exception indicates that an attempt was made to send a request using an
+    /// <see cref="IRequestHandler"/> that is no longer valid.</summary>
+    public class InvalidRequestHandlerException : Exception
     {
-        /// <summary>Constructs a new instance of the <see cref="RetryException"/> class with a reference to the inner
+        /// <summary>Constructs a new instance of the <see cref="InvalidRequestHandlerException"/> class with a reference to the inner
         /// exception that is the cause of this exception.</summary>
         /// <param name="innerException">The exception that is the cause of the current exception.</param>
-        internal RetryException(Exception innerException)
+        internal InvalidRequestHandlerException(Exception innerException)
             : base("", innerException)
         {
         }
     }
+
+    /// <summary>This exception wraps an incoming response frame containing a failure an unmarshals is retry policy,
```

Comment:
The comment isn't clear. Where is this exception actually used?

Reply:
This was used in my first version, I will remove it.